apps/website/.vuepress/theme/components/NavToc.vue

```diff
@@ -25,6 +25,31 @@ export default {
       return this.$route.path === '/foundation/icons/';
     },
   },
+  mounted: function () {
+    if (document) {
+      let headers = [...document.querySelectorAll('h2, h3')];
+      let active;
+
+      const content = document.querySelector('.content-area');
+      content.addEventListener('scroll', function () {
```

Comment:
We probably want to throttle this (not a blocker now). Besides that, some issues I see here is that: clicking on the links to activate is conflicting with this scroll spy as there is a scroll margin in `route-guard.js`. Reaching the page bottom is not activating the last link in TOC.

Reply:
I can comment this out until we are ready to fix it up a little more.

Reply:
Yeah. I think it's better to ship it once we make it more robust.